pandas/core/groupby/generic.py

```diff
@@ -429,7 +429,32 @@ def _aggregate_named(self, func, *args, **kwargs):
 
         return result
 
-    @Substitution(klass="Series")
+    __examples_series_doc = dedent(
```

Comment:
The breaking up the examples looks good - can you add an example where the argument is a string (e.g. "mean") to each one.

Reply:
Sure, this was added here: 90cae51f589f4a69d41a52308753f81ced25c76d